<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dibujarBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Dibujar</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numSidesSpinBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sideSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000000000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelScala">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>430</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Escala</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
